README.md

```diff
@@ -32,39 +32,27 @@ Rscript -e "rmarkdown::render('docs/proposal/capstone-proposal.Rmd')"
 
 ### Usage
 
-Describe data being used (links), and how to run. Eg:
-
-There are four core scripts that are required for the analysis, they
-need to be run in order:
-
--   01_clean.R
--   02_analysis.R
--   03_visualize.R
--   04_output.R
-
-#### Example
-
-This is a basic example which shows you how to solve a common problem:
-
-``` r
-## basic example code
-```
-
-### Project Status
-
-### Getting Help or Reporting an Issue
-
-To report bugs/issues/feature requests, please file an
-[issue](https://github.com/bcgov/opho-cdr-shiny/issues/).
-
-### How to Contribute
-
-If you would like to contribute, please see our
-[CONTRIBUTING](CONTRIBUTING.md) guidelines.
-
-Please note that this project is released with a [Contributor Code of
-Conduct](CODE_OF_CONDUCT.md). By participating in this project you agree
-to abide by its terms.
+Our data product is currently available for internal use only. Please contact the CDR Working Group to request access to the data.
+
+#### Temporal Modelling
+
+To re-run the analysis, please ensure that R (version 4.2.0) and RStudio are installed, then follow these instructions:
+
+1. Clone this Github repository.
+2. Create a folder named "data" in the root directory of the repository. Download and save the "Data_T_CHSA" inside this "data" folder. 
```

Comment:
We can leave an empty "data" folder in the repo root, which will help reducing one step.